packages/editor/src/components/url-popover/style.scss

```diff
@@ -9,19 +9,53 @@
 		flex-grow: 1;
 	}
 
+	// Mimic toolbar component styles for the icons in this popover
+	.components-icon-button {
+		padding: 3px;
+
+		> svg {
+			padding: 5px;
+			border-radius: $radius-round-rectangle;
+			height: 30px;
+			width: 30px;
+		}
+
+		&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover {
```

Comment:
Another note: This is a rather unfortunate string of `:not` conditions, but it's needed to override the existing styles on the standard IconButton. See the component styles here for reference:

https://github.com/WordPress/gutenberg/blob/14374e5f08ff7cd91d281cbfc4102cec39d45ed8/packages/components/src/icon-button/style.scss#L28-L30